forms/OptionsetField.php

```diff
@@ -102,4 +102,19 @@ public function validate($validator) {
 	public function ExtraOptions() {
 		return new ArrayList();
 	}
+	
+	public function getAttributes() {
+		$attributes = parent::getAttributes();
+		unset($attributes['name']);
+		if(isset($attributes['required'])) {
+			unset($attributes['required']);
+		}
+		if(isset($attributes['role'])) {
+			unset($attributes['role']);
+		}
+		if(isset($attributes['aria-required'])) {
```

Comment:
looking into this a little deeper, according to the w3 spec: *Attribute aria-required is unnecessary for elements that have attribute required.*

So aria-required seems to be now superfluous. thus the simple fix here: https://github.com/torleif/silverstripe-framework/commit/31fe519ec3b3d7d05cc422e7f574552127f7e007

should be able to resolve the required issue 

Note the required element only has to be on the first element, and I've done it for all. I could add a <% if First %> ... on there too.